### PR TITLE
Update demonstrate-deployment-on-vm.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note about hardware best practices for VM

Reference https://learn.microsoft.com/en-us/mem/autopilot/autopilot-device-guidelines

Fixes # https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10459

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md [hunk 1]
```diff
@@ -47,6 +47,9 @@ You'll need the following components to complete this lab:
 |**Hyper-V or a physical device running Windows 10**|The guide assumes that you'll use a Hyper-V VM, and provides instructions to install and configure Hyper-V if needed. To use a physical device, skip the steps to install and configure Hyper-V.|
 |**An account with Azure Active Directory (Azure AD) Premium license**|This guide will describe how to get a free 30-day trial Azure AD Premium subscription that can be used to complete the lab.|
 
+> [!NOTE]
+> When using a VM for Autopilot testing, assign at least 2 processors and 4gb of memory. *Note: The [minimum system requirements](/windows/whats-new/windows-11-requirements#virtual-machine-support) for Windows 11 are 2 processors and 4gb memory.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-autopilot/demonstrate-deployment-on-vm.md:51]:
```suggestion
> When using a VM for Autopilot testing, assign at least two processors and 4 GB of memory.
```

**Review by JohanFreelancer9 — COMMENTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@aczechowski Please merge this PR. Thanks

**Review by aczechowski — APPROVED** [review]

**Pull request merged**